scale/recipe/models.py

```diff
@@ -189,230 +141,6 @@ def create_recipe_v6(self, recipe_type_rev, event_id=None, ingest_id=None, input
 
```

Comment:
Should we remove code to copy legacy workspace info on line 121?  Assuming we convert legacy recipes to v6 input data with v6 configs for workspace information we can ditch this.  If we want to keep it so we don't have to do so much updating, we need to make sure we have the get_recipe_data method.

Reply:
I've got an issue (#1551) to create conversions as possible for all configs/definitions. I think it should be safe to say we won't be dealing with legacy config/data.